### ISSUE TITLE
STOCHRSI = 0.00000000

### ISSUE THREAD

**Issue by BitRacer** [taniman/profit-trailer#1611]
Not sure why by for a lot of major coins the STOCHRSI = 0.00000000

To help reproduce, I'm providing my indicators.
#####NEW SETTINGS for 2.0#####
SOM_trigger_length = 288

BB_std = 2
BB_candle_period = 300
BB_length = 40

SMA_cross_candles = 2
SMA_candle_period = 300
SMA_fast_length = 12
SMA_slow_length = 24

EMA_cross_candles = 3
EMA_candle_period = 300
EMA_fast_length = 7
EMA_slow_length = 34

RSI_candle_period = 1800
RSI_length = 14
STOCH_length = 14
MACD_candle_period = 300
MACD_fast_length = 12
MACD_slow_length = 26
MACD_signal = 9
OBV_candle_period = 300
OBV_length = 5
OBV_signal = 1

#OBV_candle_period = 300
#OBV_length = 5
#OBV_signal = 1
#BB_std = 0
#RSI_candle_period = 300
#RSI_length = 14
#STOCH_length = 0
#MACD_candle_period = 300
#MACD_fast_length = 0
#MACD_slow_length = 0
#MACD_signal = 0
#------------------------
#BB_period = 1800
#BB_sma = 40
#SMA_candle_period = 300
#SMA_slow_length = 24
#SMA_fast_length = 12
#EMA_period = 300
#EMA_1 = 34
#EMA_2 = 7

**Comment by SgtBatten** [comment]
I believe stochrsi does not use this parameter. But just for testing purposes, add the missing STOCH_candle_period

**Comment by BitRacer** [comment]
STOCH_candle_period = 1800 appears to have not impact.

**Comment by BitRacer** [comment]
I tried this it did not make a difference.

**Comment by sas7085** [comment]
I  was using  300 & 900 candle period i observed this  0000000  will be there few secs then the value will be appeared.

**Comment by Ghostdoge** [comment]
Yeah, I was told by support today that StochRSI will be fixed for next update.

**Comment by XcidTrader** [comment]
I prefer a shorter RSI time frame.

**Comment by hgiddy** [comment]
Guys any idea when this will be fixed please??
